tests/testthat/test-utils-html.R

```diff
@@ -0,0 +1,49 @@
+context("utils-html tests")
+
+wd <- getwd()
+setwd(dir = tempdir())
+
+test_that('does it replace the escaped characters with real html breaks?', {
+  library(jsonlite)
+  readings <- fromJSON('
```

Comment:
FYI, as you have it written, the json is actually a "reportObject" and not a "readings" object because you have the top level nesting in the json. If you want just the json for a readings object you can drop the `{ "readings":` from the start and parse just the array. Then it would be a readings (list) object being parsed.